packages/styled-components/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@essex/styled-components",
+	"description": "ESM export support for styled-components",
+	"version": "0.0.1",
+	"type": "module",
+	"main": "src/index.ts",
+	"publishConfig": {
+		"main": "dist/index.js",
+		"types": "dist/index.d.ts"
+	},
+	"sideEffects": false,
+	"scripts": {
+		"clean": "essex clean",
+		"build": "essex build --skipExportCheck --docs",
```

Comment:
This should work without --skipExportCheck

Reply:
In general, let's try to remove --skipExportCheck where we can. We may find that some other dependencies make that impossible though.